core/assets/bundles/bundle.properties

```
@@ -232,7 +232,7 @@ editor.removeunit = Remove Unit
 editor.teams = Teams
 editor.errorload = Error loading file:\n[accent]{0}
 editor.errorsave = Error saving file:\n[accent]{0}
-editor.errorimage = That's an image, not a map. Don't go around changing extensions expecting it to work.\n\nIf you want to import a legacy map, use the 'import legacy map' button in the editor.
+editor.errorimage = That's an image, not a map. Don't go around changing extensions expecting it to work.\n\nIf you want to import a legacy map, use the 'Import Legacy Map' button in the editor.
```

Gardons les changements fait a ce fichier hors de cette pr
```suggestion
editor.errorimage = That's an image, not a map. Don't go around changing extensions expecting it to work.\n\nIf you want to import a legacy map, use the 'import legacy map' button in the editor.
```

---

Sounds fair, Okay!